### PR TITLE
Update split-markets-miners.md

### PR DESCRIPTION
**Pull request by jennijuju**
Adding warning

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/split-markets-miners.md`:
- A comma may be missing after the conjunctive/linking adverb ‘Currently’.
  ```
  ... subsystems in the lotus-miner process. Currently there are 4 subsystems, th...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...istinct types of lotus-miner nodes: The markets node - a lotus-miner process...
                                             ^^^^^^^
  ```
- The preposition ‘for’ seems more likely in this position.
  ```
  ...ode - a lotus-miner process responsible to handling the storage market subsy...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...ode is started with configuration for a markets node, it includes a libp2p n...
                                             ^^^^^^^
  ```
- The word “backup” is a noun. The verb is spelled with a space.
  ```
  ... the monolith miner process, you should backup your miner's metadata reposit...
                                             ^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...the LOTUS_BACKUP_BASE_PATH env variable in order to do that. # Splitting the...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ed to create a seed config.toml for the markets node, and have it ready for ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ve it ready for the next step. For more information see configuration usage ...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...proving node on the same machine as the markets node, make sure that their l...
                                             ^^^^^^^
  ```
- Did you mean: “By default,”?
  ```
  ...sure that their listeners do not clash. By default the lotus-miner API serve...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ... you adjust the [Libp2p] section on the markets node accordingly - it needs ...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...itialising a markets service repository Create authentication tokens for the...
                                             ^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ry Create authentication tokens for the markets node. This is an online oper...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...se you move its API to another IP:PORT) Initialise the market node. This per...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  .... This performs a one-time setup of the markets node. Part of that setup inc...
                                             ^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...f that setup includes updating the peer id in the miner actor by submitting ...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...able! This command should be run on the markets miner instance host, as it i...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...er instance host, as it is creating the markets miner instance repository, a...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...aling/proving miner process without the markets subsystem Update your config...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...oving miner process without the markets subsystem Update your config.toml an...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...TH environment variable! # 6. Start the markets miner process with the marke...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...tart the markets miner process with the markets subsystem # Interacting with...
                                             ^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...rocess with the --call-on-markets flag. In order to take advantage of this f...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...he mining/sealing/proving miner and the markets miner, in your run-commands ...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ..., for example: Now that you have both a markets miner process and a mining/s...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...miner info Get the peer identity of the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...r identity of the markets miner process Get a list of storage deals from the...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...ss Get a list of storage deals from the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ge deals from the markets miner process Get a list of sectors from the minin...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...cly exposed, as we will be enabling the markets subsystem on it. In the mini...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ml and set EnableMarkets option to true Move back the DAG store directory to...
                                             ^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s it was changed to the identity of the markets node during the initialising...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...g/proving node, which now also runs the markets subsystem, i.e. currently al...
                                             ^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/split-markets-miners.md`:
- A comma may be missing after the conjunctive/linking adverb ‘Currently’.
  ```
  ... subsystems in the lotus-miner process. Currently there are 4 subsystems, th...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...istinct types of lotus-miner nodes: The markets node - a lotus-miner process...
                                             ^^^^^^^
  ```
- The preposition ‘for’ seems more likely in this position.
  ```
  ...ode - a lotus-miner process responsible to handling the storage market subsy...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...ode is started with configuration for a markets node, it includes a libp2p n...
                                             ^^^^^^^
  ```
- The word “backup” is a noun. The verb is spelled with a space.
  ```
  ... the monolith miner process, you should backup your miner's metadata reposit...
                                             ^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...the LOTUS_BACKUP_BASE_PATH env variable in order to do that. # Splitting the...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ed to create a seed config.toml for the markets node, and have it ready for ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ve it ready for the next step. For more information see configuration usage ...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...proving node on the same machine as the markets node, make sure that their l...
                                             ^^^^^^^
  ```
- Did you mean: “By default,”?
  ```
  ...sure that their listeners do not clash. By default the lotus-miner API serve...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ... you adjust the [Libp2p] section on the markets node accordingly - it needs ...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...itialising a markets service repository Create authentication tokens for the...
                                             ^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ry Create authentication tokens for the markets node. This is an online oper...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...se you move its API to another IP:PORT) Initialise the market node. This per...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  .... This performs a one-time setup of the markets node. Part of that setup inc...
                                             ^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...f that setup includes updating the peer id in the miner actor by submitting ...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...able! This command should be run on the markets miner instance host, as it i...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...er instance host, as it is creating the markets miner instance repository, a...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...aling/proving miner process without the markets subsystem Update your config...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...oving miner process without the markets subsystem Update your config.toml an...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...TH environment variable! # 6. Start the markets miner process with the marke...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...tart the markets miner process with the markets subsystem # Interacting with...
                                             ^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...rocess with the --call-on-markets flag. In order to take advantage of this f...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...he mining/sealing/proving miner and the markets miner, in your run-commands ...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ..., for example: Now that you have both a markets miner process and a mining/s...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...miner info Get the peer identity of the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...r identity of the markets miner process Get a list of storage deals from the...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...ss Get a list of storage deals from the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ge deals from the markets miner process Get a list of sectors from the minin...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...cly exposed, as we will be enabling the markets subsystem on it. In the mini...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ml and set EnableMarkets option to true Move back the DAG store directory to...
                                             ^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s it was changed to the identity of the markets node during the initialising...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...g/proving node, which now also runs the markets subsystem, i.e. currently al...
                                             ^^^^^^^
  ```

File: docs/mine/lotus/split-markets-miners.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 ## Background
 
-> Note: this feature is available starting with Lotus v1.11.1
+::: warning
+This feature is avaiable in lotus v1.11.1-rc and up, however, it is still experimental and under testing. Please use at your own risk.
+::: warning
```
Comment (jacobheun) [inline docs/mine/lotus/split-markets-miners.md:15]: [Looks like](https://v1.vuepress.vuejs.org/guide/markdown.html#default-title) only `:::` is used to close the warning block. Right now the [whole page](https://deploy-preview-988--filecoin-docs.netlify.app/mine/lotus/split-markets-miners/#frontmatter-title) is wrapped in a nested warning. 
```suggestion
:::
```

**Review by jacobheun — APPROVED** [review]
Need to fix the warning wrapper, but otherwise looks good.

**Comment by jennijuju** [comment]
> Need to fix the warning wrapper, but otherwise looks good.

lolol might as well lolol

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/split-markets-miners.md`:
- A comma may be missing after the conjunctive/linking adverb ‘Currently’.
  ```
  ... subsystems in the lotus-miner process. Currently there are 4 subsystems, th...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...istinct types of lotus-miner nodes: The markets node - a lotus-miner process...
                                             ^^^^^^^
  ```
- The preposition ‘for’ seems more likely in this position.
  ```
  ...ode - a lotus-miner process responsible to handling the storage market subsy...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...ode is started with configuration for a markets node, it includes a libp2p n...
                                             ^^^^^^^
  ```
- The word “backup” is a noun. The verb is spelled with a space.
  ```
  ... the monolith miner process, you should backup your miner's metadata reposit...
                                             ^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...the LOTUS_BACKUP_BASE_PATH env variable in order to do that. # Splitting the...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ed to create a seed config.toml for the markets node, and have it ready for ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ve it ready for the next step. For more information see configuration usage ...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...proving node on the same machine as the markets node, make sure that their l...
                                             ^^^^^^^
  ```
- Did you mean: “By default,”?
  ```
  ...sure that their listeners do not clash. By default the lotus-miner API serve...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ... you adjust the [Libp2p] section on the markets node accordingly - it needs ...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...itialising a markets service repository Create authentication tokens for the...
                                             ^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ry Create authentication tokens for the markets node. This is an online oper...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...se you move its API to another IP:PORT) Initialise the market node. This per...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  .... This performs a one-time setup of the markets node. Part of that setup inc...
                                             ^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...f that setup includes updating the peer id in the miner actor by submitting ...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...able! This command should be run on the markets miner instance host, as it i...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...er instance host, as it is creating the markets miner instance repository, a...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...aling/proving miner process without the markets subsystem Update your config...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...oving miner process without the markets subsystem Update your config.toml an...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...TH environment variable! # 6. Start the markets miner process with the marke...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...tart the markets miner process with the markets subsystem # Interacting with...
                                             ^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...rocess with the --call-on-markets flag. In order to take advantage of this f...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...he mining/sealing/proving miner and the markets miner, in your run-commands ...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ..., for example: Now that you have both a markets miner process and a mining/s...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...miner info Get the peer identity of the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...r identity of the markets miner process Get a list of storage deals from the...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...ss Get a list of storage deals from the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ge deals from the markets miner process Get a list of sectors from the minin...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...cly exposed, as we will be enabling the markets subsystem on it. In the mini...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ml and set EnableMarkets option to true Move back the DAG store directory to...
                                             ^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s it was changed to the identity of the markets node during the initialising...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...g/proving node, which now also runs the markets subsystem, i.e. currently al...
                                             ^^^^^^^
  ```

File: docs/mine/lotus/split-markets-miners.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 ## Background
 
-> Note: this feature is available starting with Lotus v1.11.1
+::: warning
+This feature is avaiable in lotus v1.11.1-rc and up, however, it is still experimental and under testing. Please use at your own risk.
```
Comment (jacobheun) [inline docs/mine/lotus/split-markets-miners.md:14]: I missed this on the first read. 🤦 
```suggestion
This feature is available in lotus v1.11.1-rc and up, however, it is still experimental and under testing. Please use at your own risk.
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/split-markets-miners.md`:
- A comma may be missing after the conjunctive/linking adverb ‘Currently’.
  ```
  ... subsystems in the lotus-miner process. Currently there are 4 subsystems, th...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...istinct types of lotus-miner nodes: The markets node - a lotus-miner process...
                                             ^^^^^^^
  ```
- The preposition ‘for’ seems more likely in this position.
  ```
  ...ode - a lotus-miner process responsible to handling the storage market subsy...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...ode is started with configuration for a markets node, it includes a libp2p n...
                                             ^^^^^^^
  ```
- The word “backup” is a noun. The verb is spelled with a space.
  ```
  ... the monolith miner process, you should backup your miner's metadata reposit...
                                             ^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...the LOTUS_BACKUP_BASE_PATH env variable in order to do that. # Splitting the...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ed to create a seed config.toml for the markets node, and have it ready for ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ve it ready for the next step. For more information see configuration usage ...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...proving node on the same machine as the markets node, make sure that their l...
                                             ^^^^^^^
  ```
- Did you mean: “By default,”?
  ```
  ...sure that their listeners do not clash. By default the lotus-miner API serve...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ... you adjust the [Libp2p] section on the markets node accordingly - it needs ...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...itialising a markets service repository Create authentication tokens for the...
                                             ^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ry Create authentication tokens for the markets node. This is an online oper...
                                             ^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...se you move its API to another IP:PORT) Initialise the market node. This per...
                                             ^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  .... This performs a one-time setup of the markets node. Part of that setup inc...
                                             ^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...f that setup includes updating the peer id in the miner actor by submitting ...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...able! This command should be run on the markets miner instance host, as it i...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...er instance host, as it is creating the markets miner instance repository, a...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...aling/proving miner process without the markets subsystem Update your config...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...oving miner process without the markets subsystem Update your config.toml an...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...TH environment variable! # 6. Start the markets miner process with the marke...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...tart the markets miner process with the markets subsystem # Interacting with...
                                             ^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...rocess with the --call-on-markets flag. In order to take advantage of this f...
                                             ^^^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...he mining/sealing/proving miner and the markets miner, in your run-commands ...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ..., for example: Now that you have both a markets miner process and a mining/s...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...miner info Get the peer identity of the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...r identity of the markets miner process Get a list of storage deals from the...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...ss Get a list of storage deals from the markets miner process Get a list of ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ge deals from the markets miner process Get a list of sectors from the minin...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...cly exposed, as we will be enabling the markets subsystem on it. In the mini...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ml and set EnableMarkets option to true Move back the DAG store directory to...
                                             ^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s it was changed to the identity of the markets node during the initialising...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...g/proving node, which now also runs the markets subsystem, i.e. currently al...
                                             ^^^^^^^
  ```

**Pull request merged**